style.css

```diff
@@ -1,3 +1,7 @@
 :root {
   font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
 }
+
+#game-container {
+  cursor: none;
```

Comment:
**suggestion**: Consider adding a comment explaining why the cursor is set to none, as this might not be immediately clear to other developers.